crates/sui/tests/transaction_orchestrator_tests.rs

```diff
@@ -157,16 +157,22 @@ async fn test_local_execution_with_missing_parents() -> Result<(), anyhow::Error
 
     let signer = context.keystore.addresses().get(0).cloned().unwrap();
     let (pkg_ref, counter_id) = publish_basics_package_and_make_counter(&context, signer).await;
+    let counter_shared_at = counter_id.1;
 
     // 0. Execute with Quorum Driver
-    let digests0 = increment(&context, &signer, counter_id, 20, pkg_ref).await;
+    let digests0 = increment(&context, &signer, counter_id.0, 20, pkg_ref).await;
```

Comment:
@longbowlu, I needed to update this test to pass in a `share_at_version`, but it was also complaining about passing an `ObjectRef` where an `ObjectID` is expected.  Is there a `From` impl I'm missing that does this...?

Reply:
I don't think so, but might be reasonable to add. 
What is here looks fine to me though

Reply:
Ah, ignore me -- I didn't realise that the first diff in the stack changed `publish_basics_package_and_make_counter` to return an `ObjectRef`, so I thought the need to introduce the `ObjectRef` to `ObjectID` conversion wasn't related to this stack and was confused how it worked to begin with.

I'll resolve this and not change the `increment`/`increment_counter` APIs, because they don't really need the whole `ObjectRef` to function.